recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "hvplot" %}
-{% set version = "0.9.2" %}
+{% set version = "0.10.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hvplot-{{ version }}.tar.gz
-  sha256: 9a8c9e9249139aaa3dee5f1cea0f93cf36374d179e986705fcddc2b92c470793
+  sha256: e87486a95bfe151ab52ef163a5e93d9cbd043992cf0b755ccadd2bf36fedd376
 
 build:
   number: 0
```

Comment:
Skip `py<38`, https://github.com/holoviz/hvplot/blob/v0.10.0/pyproject.toml#L14

Reply:
Thanks for the catch, updated in https://github.com/AnacondaRecipes/hvplot-feedstock/pull/11/commits/6d0717dfdc0ca03301dc2a6a26cd21681539b2e0